src/logger/loggerApi.c

```diff
@@ -307,6 +307,13 @@ int api_sampleData(Serial *serial, const jsmntok_t *json){
 	return API_SUCCESS_NO_RETURN;
 }
 
+int api_heartBeat(Serial *serial, const jsmntok_t *json){
```

Comment:
Dude... the longer we put off making our style consistent the harder it will be and the more likely it will be that it won't happen.  Changes don't happen overnight.  We all gotta start doing it the right way and it will encourage us to complete the migration.  Please reconsider.